### DISCUSSION TITLE
Intel Build: Could someone test this intel build on an intel mac with macos 13+?

### DISCUSSION THREAD

**Discussion by jasonjmcghee** [jasonjmcghee/rem#72] [Announcements]
I built an intel build, but don't have an intel mac (except a very outdated one).

Could someone try out [this intel build](https://github.com/jasonjmcghee/rem/releases/download/v0.1.11/rem-0.1.11-intel.dmg) and see if it works _at all_?

I assume there will be required changes to how we build ffmpeg as it will likely try to use the gpu rather than software. I assume this will drain battery life... I tried doing sw only rendering and there was some serious video artifacts whenever I _did_ anything with the computer while rem was running.

**Comment by cori** [comment]
I'm running on a 2019 16" MBP

<img width="475" alt="image" src="https://github.com/jasonjmcghee/rem/assets/46317/2e899f83-30d9-4941-b66b-4a41e5aaa731">

seems to be working fine so far; are there specific steps to build it - I'm running a from-source version on a silicon MBP and would love to do the same here

**Reply by jasonjmcghee** [reply]
You should be able to just check out the Intel branch. Ideally this would be done in build script driven way, so it doesn’t need a separate branch, but that’s the state of the world currently.
